content/docs/user-guide/interactivity/input/using-player-input.md

```diff
@@ -0,0 +1,145 @@
+
+### Using input in Lua 
+
+You can also create a Lua script that connects to Input Events. For more information about scripting with Lua, refer to [Writing Lua Scripts](/docs/user-guide/scripting/lua).
+
+1. In the viewport or **Entity Outliner**, select the entity.
+
+1. In Entity Inspector, click **Add Component**, and add the **Lua Script** component.
+
+1. In the Lua Script component, click the **Open in Lua Editor** icon.
+
+1. In the Lua Editor, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an Input Event you would like to connect the script to.
+
+1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the Input Event, and the second parameter set to create an `InputEventNotificationId` for the **Event Name**.
+
+### Example Lua script
+
+The following Lua script connects `Action` events to the individual functions `OnPressed`, `OnHeld`, and `OnReleased`.
+
+```lua
+local tutorial_input = {
+    Properties = {}
+}
+function tutorial_input:OnActivate()
+    self.ActionInputs = {}
+
+    self.ActionInputs.OnPressed = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 2.0)
+    end
+
+    self.ActionInputs.OnHeld = function(_, value)
+        TransformBus.Event.RotateAroundLocalZ(self.entityId, 0.01)
+    end
+
+    self.ActionInputs.OnReleased = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 1.0)
+    end
+
+    self.ActionInputHandler = InputEventNotificationBus.Connect(self.ActionInputs, InputEventNotificationId("Action"))
+end
+
+function tutorial_input:OnDeactivate()
+    self.ActionInputHandler:Disconnect()
+end
+
+return tutorial_input
+```
+
+## Testing player input
+
+After you use either Script Canvas or Lua to connect to an Input Handler for the Input Event in your `.inputbindings` file, you can test player input in **O3DE Editor**.
+
+1. In the viewport or **Entity Outliner**, select your entity.
+
+    {{< note >}}
+In this example, the entity must have a component that enables visibility in the game, such as the [Mesh component](/docs/user-guide/components/reference/atom/mesh), the [White Box component](/docs/user-guide/components/reference/shape/white-box), or one of the [Shape components](/docs/user-guide/components/reference/shape).
+{{< /note >}}
+
+1. In Entity Inspector, click **Add Component**, and add the **Mesh** component.
+
+1. For **Mesh asset**, specify a mesh asset file. This gives your entity a shape that is visible in game mode.
+
+1. Press **Ctrl+G** to enter game mode.
+
+1. Press the keyboard **Spacebar** so that your entity rotates on the local z-axis. The entity's scale doubles when the event `OnPressed` is received from the Input Handler that is connected to `Action`.  The entity's scale returns to normal when the `OnReleased` event is received.
```

Comment:
```suggestion
1. Press **Space bar** so that your entity rotates on the local z-axis. The entity's scale doubles when the event `OnPressed` is received from the input handler that is connected to `Action`. The entity's scale returns to normal when the `OnReleased` event is received.
```
To avoid passive voice ("is received"), can we identify who/what receives the events? For example:
>The entity's scale doubles when [noun] receives the `OnPressed` event from the input handler that is connected to `Action`. The entity's scale returns to normal when [noun] receives the `OnReleased` event.

Reply:
Thanks for mentioning this too, I was unsure how to treat Script Canvas nodes. The capitalization of Input Handler was mean to convey that it is a node.  I'll correct the passive voice; any thoughts on formatting for SC nodes though?